packages/e2e-tests/tests/file-search-01.test.ts

```diff
@@ -29,23 +29,23 @@ test("file-search-01: should search files", async ({ pageWithMeta: { page, recor
   // Verify search results for the string "test"
   await searchSources(page, "test");
   await verifySourceSearchOverflowMessageShown(page, false);
-  await verifySourceSearchSummary(page, "21 results in 1 file");
-  await verifyVisibleResultsCount(page, 25); // 21 results in 4 different files
+  await verifySourceSearchSummary(page, "12 results in 1 file");
+  await verifyVisibleResultsCount(page, 16); // 12 results in 4 different files
```

Comment:
Something looks broken here. There are 12 results in 4 files, but the search summary says 12 results in 1 file.

![Screenshot 2024-01-18 at 9 56 47 AM](https://github.com/replayio/devtools/assets/29597/75a280fb-105e-4212-9bb0-1612069bc9c9)